smithy-base/src/main/java/software/amazon/smithy/gradle/tasks/SmithyBuildTask.java

```diff
@@ -104,9 +118,20 @@ List<String> getModelAbsolutePaths() {
                 .collect(Collectors.toList());
     }
 
+    /**
+     * Read-only property.
+     *
+     * @return Returns true if at least one of the specified build configs exists.
+     */
+    @Internal
+    Provider<Boolean> getSmithyBuildConfigsExist() {
+        return getSmithyBuildConfigs().map(files -> !files.filter(File::exists).isEmpty());
```

Comment:
Does this allow setting invalid file locations for configs so long as noBuildConfig is set to true?